src/elements/content-preview/ContentPreview.js

```diff
@@ -339,6 +339,8 @@ class ContentPreview extends React.PureComponent<Props, State> {
             this.loadPreview();
         } else if (hasTokenChanged) {
             this.updatePreviewToken();
+            this.api.updateToken(token);
```

Comment:
Should this be supported for all top-level Elements? It doesn't seem preview-specific.

Reply:
This came about because having the knobs in the storybook to switch tokens through props. I'm not sure other elements have this issue, but there is another strategy I can do inside storybook that will basically refresh reload the component with a button when we change the token and file id's